app.js

```diff
@@ -571,4 +571,31 @@ document.addEventListener("DOMContentLoaded", () => {
     // Toggle the visibility of the <p> tag based on whether there are challenges
     noMatchingChallengesMsg.style.display = challengesDiv && challengesDiv.children.length === 0 ? "block" : "none";
   }
-});
\ No newline at end of file
+});
+
+//API booking part two
+
+
+
+async function updateParticipantsDisplay(id) {
+  const api = new APIadapter();
+  const challengeCard = await api.fetchAllChallenges();
```

Comment:
This variable name can be confusing. It sounds to me like the name `challengeCard` would be used for a variable that contains a single thing (because "Card" is singular, not plural) but it doesn't. It contains a list of many challenges (returned by `api.fetchAllChallenges()`).